README.md

```diff
@@ -1,223 +1,113 @@
+[OpenSSL PR 8797](https://github.com/openssl/openssl/pull/8797).
+We will endeavor to track OpenSSL releases within a day or so, and there is an
+item below about how we'll follow their tagging.
 
-For Testing and Development
----------------------------
+On to the questions and answers.
 
-Although testing and development could in theory also be done using
-the source tarballs, having a local copy of the git repository with
-the entire project history gives you much more insight into the
-code base.
+What about branches?
+--------------------
 
-The official OpenSSL Git Repository is located at [git.openssl.org].
-There is a GitHub mirror of the repository at [github.com/openssl/openssl],
-which is updated automatically from the former on every commit.
+We don't want to conflict with OpenSSL branch names. Our current plan is to append
+`+quic`. Release tags are likely to be the QUIC branch with `-releaseX` appended.
+For example, the OpenSSL tag `openssl-3.0.0-alpha12` would have a branch named
+`openssl-3.0.0-alpha12+quic` and a release tag of `openssl-3.0.0-alpha12+quic-release1`
 
-A local copy of the Git Repository can be obtained by cloning it from
-the original OpenSSL repository using
+How are you keeping current with OpenSSL?
+-----------------------------------------
 
-    git clone git://git.openssl.org/openssl.git
+(In other words, "What about rebasing?")
 
-or from the GitHub mirror using
+Our plan is to always rebase on top of an upstream release tag. In particular:
 
-    git clone https://github.com/openssl/openssl.git
+- The changes for QUIC will always be at the tip of the branch -- you will know what
+  is from the original OpenSSL and what is for QUIC.
+- New versions are quickly created once upstream creates a new tag.
+- The use of git commands (such as "cherry") can be used to ensure that all changes
 
-If you intend to contribute to OpenSSL, either to fix bugs or contribute
-new features, you need to fork the OpenSSL repository openssl/openssl on
-GitHub and clone your public fork instead.
+have moved forward with minimal or no changes. You will be able to see
+QUIC: Add X" on all branches and the commit itself will be nearly identical on all
+branches, and any changes to that can be easily identified.
 
-    git clone https://github.com/yourname/openssl.git
+What about library names?
+-------------------------
 
-This is necessary, because all development of OpenSSL nowadays is done via
-GitHub pull requests. For more details, see [Contributing](#contributing).
+Library names will be the same, but will use a different version number. The version
+numbers for the current OpenSSL libraries are `1.1` (for the 1.1.0 and 1.1.1 branches)
+and `3` (for the to-be-3.0 branch). We will be prefixing 81 (ASCII for 'Q') to
+the version numbers to generate a unique version number.
 
-Build and Install
-=================
+- libcrypto.so.81.3 vs libcrypto.so.3
+- libcrypto.so.81.1.1 vs libcrypto.so.1.1   libcrypto.so.3
```

Comment:
(likewise the stray "libcrypto.so.3" could be removed in a follow-up)

Reply:
Fixed in tip commit